she_codes_news/news/urls.py

```diff
@@ -4,5 +4,10 @@
 app_name = 'news'
 
 urlpatterns = [
-    path('', views.IndexView.as_view(), name='index')
+    path('', views.IndexView.as_view(), name='index'),
+    path('<int:pk>/', views.StoryView.as_view(), name='story'),
+    path('add-story/', views.AddStoryView.as_view(), name='newStory'),
+    path('edit-story/<int:pk>/', views.UpdatePostView.as_view(), name='update_post'),
```

Comment:
i probably would have used <int:pk>/edit/ and <int:pk>/delete/
Rather than edit-story/<int:pk>/